index.py

```diff
@@ -106,7 +106,8 @@ def embeddings(self, text):
 
 
 client_mode = os.getenv("CLIENT_MODE")
-ai_client = AIClientAdapter(client_mode)
+ollama_url = os.getenv("OLLAMA_ENDPOINT")
```

Comment:
Also, this should be defaulting to `http://localhost:11434` incase the env variable is not set